app/views/teams/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<h3>Update Team Details</h3>
+<%= form_with model: @team do |form| %>
```

Comment:
If the form is the same in both edit and new, then move the form to `_form.html.erb`. Look at projects or skills form if u need an example.